src/main/java/com/revature/chronicle/controller/TicketController.java

```diff
@@ -0,0 +1,202 @@
+package com.revature.chronicle.controller;
+
+import java.sql.Date;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.chronicle.models.Notification;
+import com.revature.chronicle.models.Ticket;
+import com.revature.chronicle.models.User;
+import com.revature.chronicle.services.NotificationService;
+import com.revature.chronicle.services.TicketService;
+
+@RestController(value="ticketController")
+@RequestMapping(path="/ticket")
+public class TicketController {
+	
+	@Autowired
+	private TicketService ticketService;
+	
+	@Autowired
+	private NotificationService notificationService;
+	
+	//this endpoint returns all notifications for the trainer
+	@GetMapping(path="notifications")
+	public ResponseEntity <List<Ticket>> getNotifications(HttpServletRequest req){
+		 User user =  (User) req.getAttribute("user");
+		 List<Ticket> tickets = ticketService.ticketsByEditor(user);
+		 return new ResponseEntity<>(tickets, HttpStatus.OK);
+	}
+
+	

```

Comment:
Can you make the resource names consistent? You shouldn't use "all" here as this really isn't the only endpoint that returns "all" of something. Following the convention of the "notifications" endpoint.

Reply:
We can make those changes Christina. Thanks for reminding us about some of the other changes you requested or questioned. As for your earlier question about the two register methods, we have decided that we do not need either one of them, so we are deleting them as we speak.